apps/src/dance/Dance.js

```diff
@@ -488,8 +488,7 @@ Dance.prototype.preview = async function () {
   const previewDraw = () => {
     this.nativeAPI.setEffectsInPreviewMode(true);
 
-    // the user setup hook initializes effects,
-    // needs to happen in preview mode for some effects (eg, tacos)
+    // calls each effect's init() step
```

Comment:
A couple of unrelated comment clarifications based on some previous PR feedback, LMK what you think @fisher-alice

Reply:
Short and clear - thanks!